circle.yml

```diff
@@ -3,44 +3,46 @@ machine:
     - redis
     - docker
   post:
-    - /usr/bin/redis-server --port 6380:
+    - /usr/bin/redis-server --port 6379:
```

Comment:
I think this needs to be `6380`. It brings up a second redis instance to run multi-instance tests (one on port 6379 is brought up by circle - see line 3 of this file)